common/etc/nginx/nginx.conf

```diff
@@ -31,6 +31,7 @@ env PROXY_CACHE_VALID_NOTFOUND;
 env PROXY_CACHE_VALID_FORBIDDEN;
 env HEADER_PREFIXES_TO_STRIP;
 env FOUR_O_FOUR_ON_EMPTY_BUCKET;
+env STRIP_LEADING_DIRECTORY_PATH;
```

Comment:
Suggestion: 
The variable "STRIP_LEADING_DIRECTORY_PATH" can be set as a toggle switch, and the PATH can be directly use "S3_BUCKET_NAME".